client/src/main/java/edu/colostate/cs/cs414/chesshireCoders/jungleClient/client/AuthTokenManager.java

```diff
@@ -20,6 +20,7 @@ public static AuthTokenManager getInstance() {
     }
 
     private AuthToken authToken;
+    private String email;
```

Comment:
Why are we adding an email to the AuthTokenManager? I think the token itself maps to a user, so you should just be able to get the nickname of the user associated with that token.